src/test/java/selenium/webdriver/ParametrizedLoginTest.java

```diff
@@ -0,0 +1,60 @@
+package selenium.webdriver;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import selenium.TestUser;
+
+import java.util.stream.Stream;
+
+import static java.time.Duration.ofSeconds;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
+import static selenium.Locators.*;
+
+public class ParametrizedLoginTest {
+    private WebDriver driver;
+    @BeforeEach
+    public void setup() {
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(ofSeconds(5));
+    }
```

Comment:
please add 1 blank line between class name and fields, between fields and methods for better readability of the code

Reply:
Done